Simplenote/Classes/SPSettingsViewController+Extensions.swift

```diff
@@ -88,3 +88,77 @@ private extension BiometricAuthentication.Biometry {
         }
     }
 }
+
+// MARK: - Account Deletion
+extension SPSettingsViewController {
+
+    @objc
+    func deleteAccountWasPressed() {
+        guard let user = SPAppDelegate.shared().simperium.user else {
+            return
+        }
+        SPTracker.trackDeleteAccountButttonTapped()
+        let pendingAlert = ActivityIndicatorAlertViewController()
+        pendingAlert.startAnimating()
+
+        let deletionController = AccountDeletionController()
+        deletionController.successHandler = {
+            pendingAlert.stopAnimating()
+            pendingAlert.dismiss(animated: false, completion: nil)
+            self.presentSuccessAlert(for: user)
+        }
+
+        deletionController.failureHandler = { status in
+            if status == 0 {
+                pendingAlert.stopAnimating()
+                pendingAlert.dismiss(animated: false, completion: nil)
+                NoticeController.shared.present(NoticeFactory.networkError())
+                return
+            }
+
+            let alert = UIAlertController(title: AccountDeletion.errorTitle, message: AccountDeletion.errorMessage, preferredStyle: .alert)
+            alert.addDefaultActionWithTitle(AccountDeletion.ok)
```

Comment:
Looks like either dead code or sad alert that never got presented

Reply:
That should get presented if the response code from the server is anything other than 2xx or 0
I am not sure that there are any cases where we would get back a not 2xx return value.

Reply:
There is no code to present it, current code only creates an alert but doesn't present it.

Reply:
Oh shoot, how did I miss that?  Thank you!!